examples/dev/trace-analytics-sample-app/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gradle:jdk14 AS builder
+FROM gradle:jdk11 AS builder
```

Comment:
Why make this change and downgrade?

Reply:
In the main branch, this example has an image build issue with jdk14, sth like
```
No method Found: java.lang.string.containsignorecase()
```

Reply:
We should be using JDK 11 as our baseline anyway.